src/code_800AF9B0.c

```diff
@@ -3213,64 +3216,69 @@ GLOBAL_ASM("asm/non_matchings/code_800AF9B0/func_800B4DCC.s")
 
 #ifdef MIPS_TO_C
 //generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-s32 func_800B4DF4(void *arg0) {
-    return ((arg0->unk1 << 8) + arg0->unk0 + (arg0->unk2 << 0x10)) & 0xFFFFFF;
+// Convert time trial record from its little-endian form to big-endian
+s32 func_800B4DF4(u8 *arg0) {
+    return (arg0[0] + (arg0[1] << 8) + (arg0[2] << 0x10)) & 0xFFFFFF;
```

Comment:
In the future this could end up as a macro something like:
#define BIG_TO_LITTLE_ENDIAN ()